crates/marketplace/src/builder_state.rs

```diff
@@ -876,10 +876,10 @@ impl<TYPES: NodeType> BuilderState<TYPES> {
         self.process_da_proposal(rda_msg).await;
     }
 
-    /// [event_loop_helper_handle_qc_proposal] is a helper function that is used
+    /// [event_loop_helper_handle_quorum_proposal] is a helper function that is used
     /// to handle incoming [MessageType]s, specifically [QuorumProposalMessage]s,
     /// that are received by the [BuilderState::qc_receiver] channel.
```

Comment:
With the "Happy Path" refactor I had introduced some new `qc` terms as I had spread them out.  These need to be adjusted.  Specifically the comment here for `BuilderState::qc_receiver`.

Reply:
Addressed in 155a08b434a16cf9bad6bb4932b82bbcdf1f5220.